source/_docs/z-wave/controllers.markdown

```diff
@@ -85,9 +85,20 @@ It's totally normal for your Z-Wave stick to cycle through its LEDs (Yellow, Blu
 
 ### Razberry Board
 
-You need to disable the on-board Bluetooth since the board requires the use of the hardware UART (and there's only one on the Pi3). You do this by adding the following to the end of `/boot/config.txt`:
+You need to disable the on-board Bluetooth since the board requires the use of the hardware UART (and there's only one on the Raspberry Pi 3). You do this by adding the following to the end of `/boot/config.txt`:
```

Comment:
There should be another way - use the `miniuart-bt` (or `pi3-miniuart-bt` on Pi3). But it's out of scope what's being changed here, so adding it just for the record.